wagtaildraftail/decorators.py

```diff
@@ -25,21 +25,27 @@ def HR(props):
 
 class Link:
     def render(self, props):
-        data = props.get('data', {})
+        link_type = props.get('linkType', '')
+        title = props.get('title')
```

Comment:
`title` needs to come from the props, not the page – otherwise all internal links would have a title by default.